packages/xchain-thornode/src/config.ts

```diff
@@ -0,0 +1,4 @@
+export const THORNODE_API_TS_URL = 'https://thornode.thorswap.net/';
+export const THORNODE_API_9R_URL = 'https://thornode.ninerealms.com/';
```

Comment:
Let users define custom urls by using `.env`. If it is not set in `.env`, then use `https://thornode.ninerealms.com` by default. All others can be removed from `config.ts`,  `https://thornode.thorchain.info` might be deprecated in the near future.

Reply:
yep sure thing. ill fix this now